react/week1/my-app/src/App.js

```diff
@@ -0,0 +1,37 @@
+//import logo from './logo.svg';
+import './App.css';
+import Header from "./Header.js";
+import ListItems from "./List.js";
+
+const toDoArray =[
+  {
+  description : "Clean the house",
+  dueDate: new Date(2020, 12, 25)
+},
+{
+  description : "Buy a new shirt",
+  dueDate: new Date(2021, 10, 20)
+},
+{description : "Finish my homework",
+  dueDate: new Date(2021, 5, 10)
+}
+]
```

Comment:
The `App` component is not really concerned with the Todos. I would move this down into the Todos component. 

There could be use case where this piece of data needs to be shared across the entire application. But in this case it is used only in the Todos component.